packages/tools/pie-wrapper-react/custom-elements.json

```diff
@@ -1,43 +0,0 @@
-{
-    "schemaVersion": "1.0.0",
-    "readme": "",
-    "modules": [
-        {
-            "kind": "javascript-module",
-            "path": "pie-wrapper-react",
```

Comment:
I'm guessing this file isn't required? There isn't much here so fair enough if so

Reply:
This was added to the .gitignore a while ago but lingered in the git cache. This correctly deletes the file from source control :)